packages/react-slider/src/components/Slider/Slider.test.tsx

```diff
@@ -45,93 +45,6 @@ describe('Slider', () => {
       const { container } = render(<Slider defaultValue={5} origin={2} vertical min={0} max={10} />);
       expect(container).toMatchSnapshot();
     });
-
-    it('renders Slider with marks correctly', () => {
```

Comment:
You should run `yarn jest -u` within `react-slider` to actually remove these snapshots from `Slider.test.tsx.snap`